src/components/TokenLive.tsx

```diff
@@ -20,7 +20,8 @@ const TokenLive = ({tokenId, width, height}: Props) => {
   const [pollingTime, setPollingTime] = useState(0)
   const [pollingDelay, setPollingDelay] = useState(0)
   const [pollingAttempts, setPollingAttempts] = useState(0)
-  const endpoint = `${GENERATOR_URL}/${CORE_CONTRACT_ADDRESS?.toLowerCase()}/${tokenId}`
+  const generatorUrl = CONTRACT_INFO[tokenId.split('-')[0]].GENERATOR_URL
+  const endpoint = `${generatorUrl}/${tokenId.split('-')[0].toLowerCase()}/${tokenId.split('-')[1]}`
```

Comment:
ditto the above, we can probs add the contract field to the GetToken query instead of splitting all these places, for readability reasons

Reply:
addressed with [7202d8b](https://github.com/ArtBlocks/artblocks-engine-react/pull/10/commits/7202d8bf8996b2b5c2c735f6bc4f98af38528f59)